ui/src/app/base/components/node/DiskNumaNodes/DiskNumaNodes.test.tsx

```diff
@@ -1,16 +1,16 @@
 import { mount } from "enzyme";
 
-import NumaNodes from "./NumaNodes";
+import DiskNumaNodes from "./DiskNumaNodes";
```

Comment:
I've renamed this to help make it clearer this is for NUMA nodes by disk and not by NIC